<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-4MKSC1R" failures="1" tests="10" name="Surefire test" time="17.133" errors="0" timestamp="2024-08-07T12:49:34 PKT">
  <testcase classname="api.test.DDTests" name="testPostUser" time="3.63"/>
  <testcase classname="api.test.DDTests" name="testPostUser" time="1.06"/>
  <testcase classname="api.test.DDTests" name="testPostUser" time="1.052"/>
  <testcase classname="api.test.DDTests" name="testDeleteUserByName" time="1.078"/>
  <testcase classname="api.test.DDTests" name="testDeleteUserByName" time="1.069">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.DDTests.testDeleteUserByName(DDTests.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
  <testcase classname="api.test.DDTests" name="testDeleteUserByName" time="1.067"/>
  <testcase classname="api.test.UserTests" name="testPostUser" time="1.067"/>
  <testcase classname="api.test.UserTests" name="testGetUserByName" time="1.074"/>
  <testcase classname="api.test.UserTests" name="testUpdateUser" time="2.116"/>
  <testcase classname="api.test.UserTests" name="testDeleteUserByName" time="1.038"/>
</testsuite> <!-- Surefire test -->
